src/com/yaji/Rope.java

```diff
@@ -2,26 +2,31 @@
 
 import java.util.Date;
 
+import static com.yaji.GenderType.NON;
+
 public class Rope extends Item{
     private int critFalls;
-    private String outputString="";
 
-    public Rope(Date purchaseDate, Date lastDate, String brand,double size, int gender, int critFalls){
-        super(purchaseDate, lastDate, brand, size, 9, "rope");
+
+    public Rope(Date purchaseDate, Date lastDate, String brand, double size, GenderType gender, int critFalls){
+        super(purchaseDate, lastDate, brand, size, NON, "rope");
```

Comment:
There is not really any reason to have the `gender` parameter in the constructor, since you anyway just pass `NON` to the `super` constructor. It therefore doesn't matter what you pass to the `Rope` constructor.